include/user.php

```diff
@@ -0,0 +1,40 @@
+<?php 
+
+function getUser($accountId){
+    $user = dbQuery('
+    SELECT *
+    FROM trackerSignUp
+    WHERE accountId = :accountId', 
+    [
+        'accountId' => $accountId
+    ])->fetch();                             
+return @$user['accountId'];
```

Comment:
I think maybe you meant to put this in `attemptLogin`. The name of this function suggests it should return the user, not the accountId